cmake/cpu_arch_flags.cmake

```diff
@@ -47,6 +47,11 @@ function(get_arch_flags architecture compiler)
             set(CPU_ARCH_FLAGS "-mcpu=thunderx2t99" PARENT_SCOPE)
         endif()
 
+    # IBM power 9
+    elseif ("${HOST_ARCH}" STREQUAL "power9")
+        set(HAS_REDZONE OFF PARENT_SCOPE)
+        set(CPU_ARCH_FLAGS "-mtune=power9" PARENT_SCOPE)
```

Comment:
Should be -march=power9 (as on all archs except on arm!). Or is this similar to arm? In this case -mcpu should be used...